bleak/backends/p4android/defs.py

```diff
@@ -87,4 +87,4 @@ class ScanFailed(enum.IntEnum):
     BluetoothGattCharacteristic.PROPERTY_WRITE_NO_RESPONSE: "write-without-response",
 }
 
-CLIENT_CHARACTERISTIC_CONFIGURATION_UUID = "2902"
+CLIENT_CHARACTERISTIC_CONFIGURATION_UUID = "00002902-0000-1000-8000-00805f9b34fb"
```

Comment:
How about using `normalize_uuid_16()` here instead of writing out the entire UUID?